hack/generate-manifest-clickhouse-tls.sh

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env bash
```

Comment:
we don't need this file and we don't need to generate the manifest. You can install the chart with Helm directly.

Reply:
Removed.